tasks/publish-pyxis-repository/publish-pyxis-repository.yaml

```diff
@@ -85,6 +88,11 @@ spec:
         fi
 
         DATA_FILE="$(workspaces.data.path)/$(params.dataPath)"
+        RESULTS_DIR="$(workspaces.data.path)/$(params.resultsDirPath)"
+        RESULTS_FILE="${RESULTS_DIR}/publish-pyxis-repository-results.json"
+        RESULTS_JSON='{"images":[]}'
```

Comment:
I'm not sure how this will behave since `images` is used here https://github.com/konflux-ci/release-service-catalog/blob/development/tasks/push-snapshot/push-snapshot.yaml#L96 too. I'd recommend checking the release cr in the e2e to make sure both tasks results are there. The update-cr-status task may overwrite one task's results with the other

Reply:
Resolving in favor of https://github.com/konflux-ci/release-service-catalog/pull/503#discussion_r1711198212